UNITTESTS/CMakeLists.txt

```diff
@@ -112,7 +112,9 @@ set(unittest-includes-base
   "${PROJECT_SOURCE_DIR}/stubs"
   "${PROJECT_SOURCE_DIR}/.."
   "${PROJECT_SOURCE_DIR}/../features"
-  "${PROJECT_SOURCE_DIR}/../platform"
+  "${PROJECT_SOURCE_DIR}/../features/netsocket"
```

Comment:
Doesn't seem to belong to this PR? Rebase problem?

Reply:
Fixed.